docs/cli.rst

```diff
@@ -1,11 +1,302 @@
+
+After installing CumulusCI, use the ``cci`` command in your terminal or command prompt to interact with it.
+
+On macOS, access the terminal via ``Terminal.app``. On Windows, open ``cmd.exe``. On Linux, use your preferred terminal application.
+
+To see all available commands, type ``cci`` in your terminal.
+
+.. code-block:: console
+
+    $ cci
+    Usage: cci [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    error    Get or share information about an error
+    flow     Commands for finding and running flows for a project
+    org      Commands for connecting and interacting with Salesforce orgs
+    project  Commands for interacting with project repository configurations
+    service  Commands for connecting services to the keychain
+    shell    Drop into a Python shell
+    task     Commands for finding and running tasks for a project
+    version  Print the current version of CumulusCI
+
+To retrieve information on a specific command, type ``cci <command>``.
+
+    Example: ``cci task``
+
+.. code-block:: console
+
+    $ cci task
+    Usage: cci task [OPTIONS] COMMAND [ARGS]...
+
+    Commands for finding and running tasks for a project
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    doc   Exports RST format documentation for all tasks
+    info  Displays information for a task
+    list  List available tasks for the current context
+    run   Runs a task
+
+The ``cci task`` command also has many useful subcommands, such as ``cci task info <name>``.
+
+
+
+Get Help Running Tasks
+----------------------
+
+If you're not certain about what a specific command does, use the ``--help`` flag to get more information. 
+
+    Example: ``cci task run --help``
+
+For top-level commands (that don't do anything on their own but have additional subcommands underneath them), this output will be the same with or without the ``--help`` flag.
+
+.. code-block::
+
+    $ cci task info <name> --help
+     Usage: cci task run [OPTIONS] TASK_NAME
+
+When the ``--help`` flag is specified for a command, the output will include:
+
+* A usage statement featuring the syntax that executes the command.
+* A description of the command.
+* The list of available options for use with the command.
+
+.. code-block:: console
+
+    $ cci task --help
+    Usage: cci task [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    doc   Exports RST format documentation for all tasks
+    info  Displays information for a task
+    list  List available tasks for the current context
+    run   Runs a task
+
+
+If you're just getting started with CumulusCI, don't worry if you aren't sure which of the many tasks and flows to use. We show you specific tasks and flows in later sections of the documentation. 
+
+
+
+List Tasks and Flows
```

Comment:
This heading does not match the section content.

Reply:
Looking at the commit history it looks like some headings were altered and this content was removed. I've added it back in my most recent commit as well as fixed the headings. fyi @justinvholt